pkg/helper/node.go

```diff
@@ -0,0 +1,16 @@
+package helper
+
+import (
+	"os"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Return true if the event name is the name of
+// the pods's node (reading the env var NODE_NAME)
+func EventIsForThisNode(meta v1.Object) bool {
```

Comment:
can we save the node that we are running as part of the controller? no need to check this on every request this is static for all the pod running life time.

Reply:
Is about reading environment variables let's not add complexity in the controller.